serverless-logic/serverless-logic-managing-services/serverless-logic-configuring-openAPI-services.adoc

```diff
@@ -0,0 +1,13 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="serverless-logic-configuring-openAPI-services_{context}"]
```

Comment:
```suggestion
[id="serverless-logic-configuring-openAPI-services"]
```

Not needed at the top assembly level. See [guideance](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata).

Reply:
Removed